### ISSUE TITLE
Image alt attributes

### ISSUE THREAD

**Issue by ghost** [THOM3121/f1-3-c2p1-colmar-academy#4]
https://github.com/THOM3121/f1-3-c2p1-colmar-academy/blob/master/Colmar/index.html#L116-L146

Consider giving your images `alt` attributes. Be consistent and use them for every image, especially if they're related to navigation or for whatever reason just need to be communicated.

This will help improve the accessibility of your website even when it's unable to render images, or when visually impaired users access your website via a reader or other device that doesn't show images.
